### ISSUE TITLE
feat: `sendImageAsSticker` should be as flexible (in terms of source media) as `sendMp4AsSticker`

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1447]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci minor`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.8.0](https://github.com/open-wa/wa-automate-nodejs/compare/3.7.0...3.8.0) (2021-03-06)

- build(deps): bump open from 7.4.2 to 8.0.2 [`#1439`](https://github.com/open-wa/wa-automate-nodejs/pull/1439)
- build(deps-dev): bump typescript from 4.2.0-beta to 4.2.3 [`#1444`](https://github.com/open-wa/wa-automate-nodejs/pull/1444)
- build(deps): bump patch-package from 6.4.4 to 6.4.5 [`#1445`](https://github.com/open-wa/wa-automate-nodejs/pull/1445)
- build(deps): bump swagger-ui-dist from 3.43.0 to 3.44.1 [`#1446`](https://github.com/open-wa/wa-automate-nodejs/pull/1446)
- 🐛  fixes #1441 and eslint improvements [`#1441`](https://github.com/open-wa/wa-automate-nodejs/issues/1441)
- Release 3.8.0 [`2209183`](https://github.com/open-wa/wa-automate-nodejs/commit/220918322cdfc999a3086ca6028dc73e384f21d3)
- ✨  `sendImageAsSticker` now accepts buffer, url, filename and base64 #1447 [`a37e819`](https://github.com/open-wa/wa-automate-nodejs/commit/a37e819542ddff3a443fa7b5335ed61fef1b3f8b)
- 🩹 sticker requests now include fallback retry. video stickers moved to gcp #1442 [`6d58763`](https://github.com/open-wa/wa-automate-nodejs/commit/6d587638a3f2550d9a63868b91178b4a50847286)
- 🔧  fix popup code for new open deps [`ed558a9`](https://github.com/open-wa/wa-automate-nodejs/commit/ed558a92b594e71952fef5f0bec4a5afc4ffe69d)
- 🥚  patched `getMyLastMessage` [`d8d6ea3`](https://github.com/open-wa/wa-automate-nodejs/commit/d8d6ea3f97226f43648c12ebea28da245dda16fa)
- 🥚  fix duplicate callbacks [`6abb8d6`](https://github.com/open-wa/wa-automate-nodejs/commit/6abb8d6c1a7c3700336f2b0f551d8aa3354c4adf)
- 🥚  #1443 [`22b2eff`](https://github.com/open-wa/wa-automate-nodejs/commit/22b2eff16d89d1c3be4cb1a904a6daac048b11a0)
- ⚡️ load `popup` only when needed #1441 [`230c199`](https://github.com/open-wa/wa-automate-nodejs/commit/230c199eb3127a157e20079fc00b1f29474b8a8f)
- 🥚  autoPreview now converts datauri to base64 automatically [`f8aa670`](https://github.com/open-wa/wa-automate-nodejs/commit/f8aa670851f271accebc216cdf2e052afe2f94de)
- 🗑 deprecated `sendMessageWithThumb`. use `sendLinkWithAutoPreview` instead [`ce72a1e`](https://github.com/open-wa/wa-automate-nodejs/commit/ce72a1ea62594890d2e2524d6f8b5a3ec445c65f)
- 🥚  patch update [`2897469`](https://github.com/open-wa/wa-automate-nodejs/commit/2897469dcf124deda590bf8a5b1899bf28703a90)
- 🥚  patch update [`7135580`](https://github.com/open-wa/wa-automate-nodejs/commit/7135580778a623ce634adbe7cfa6bf04e55a9fc0)
- 🥚  patch update [`63aefb8`](https://github.com/open-wa/wa-automate-nodejs/commit/63aefb8659c90275a63d28e3d4d116a251394efd)
